.github/workflows/platformDeploy.yml

```diff
@@ -238,6 +245,13 @@ jobs:
           name: ios-sourcemap
           path: main.jsbundle.map
 
+      - name: Upload iOS version to GitHub artifacts
+        if: ${{ !fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: New Expensify.ipa
+          path: /Users/runner/work/App/App/New Expensify.ipa
```

Comment:
How did you determine this path? Curious why it's an absolute path out of `/Users/`, as opposed to the relative path for android.

Reply:
I just copied the paths from the step below, which uploads the files to browserstack